### PR TITLE
allgemein: fix gender

### PR DESCRIPTION
**Pull request by jnccd**
Setzt die Guidelines um, die intern bespochen wurden

### CODE REVIEW

File: hyperbook/book/allgemein/index.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ Das Anmeldeverfahren sieht etwa wie folgt aus. Genaue Termine des
 aktuellen Zeitraumes finden sich auf der [Website unter
 Ablauf und Termine](https://software-challenge.de/ablauf-und-termine).
 
-Ab April können interessierte SchulleiterInnen, Lehrkräfte und
-SchülerInnen ihre Schulen und Teams für die Teilnahme an der
+Ab April können interessierte Schulleiterinnen, Schulleiter, Lehrkräfte,
```
Comment (FalconsSky) [inline hyperbook/book/allgemein/index.md:53]: Wouldn't it make sense to just name the *Lehrkräfte* and the *SuS*? After all, the principals are usually teachers, too.

**Review by FalconsSky — COMMENTED** [review]
The rest looks good to me.

File: hyperbook/book/allgemein/index.md (hunk 1 above)
Comment (jnccd) [reply hyperbook/book/allgemein/index.md:53]: Fair, I just directly adapted the text here, but I might as well change that

Comment (jnccd) [reply hyperbook/book/allgemein/index.md]: sus

**Review by FalconsSky — APPROVED** [review]
Looks good.

**Pull request merged**